elodie/compatability.py

```diff
@@ -1,7 +1,31 @@
+import os
+
 def _decode(string, encoding='utf8'):
     if hasattr(string, 'decode'):
         return string.decode(encoding)
 
     return string
 
 
+def _copyfile(src, dst):
+    try:
+        O_BINARY = os.O_BINARY
+    except:
+        O_BINARY = 0
+
+    READ_FLAGS = os.O_RDONLY | O_BINARY
+    WRITE_FLAGS = os.O_WRONLY | os.O_CREAT | os.O_TRUNC | O_BINARY
+    BUFFER_SIZE = 128*1024
```

Comment:
Can we include this optimization? 128k still seems small. This might increase the performance even more.

```
        TEN_MEGABYTES = 10485760
        BUFFER_SIZE = min(TEN_MEGABYTES, os.path.getsize(source))
```